CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.2.6] - 2019-09-13
+
+### Changed
+
+- Updated version of Mobile Developer Console
```

Comment:
Should there also be a point here about updating resource memory and CPU allocation changes?